facia/app/utils/FaciaPicker.scala

```diff
@@ -8,24 +8,39 @@ import play.api.mvc.RequestHeader
 
 class FaciaPicker extends GuLogging {
 
-  // we don't yet use these parameters but they will play into dcrCouldRender later on
+  val SUPPORTED_COLLECTIONS: Set[String] =
+    Set("dynamic/fast", "dynamic/slow", "fixed/small/slow-IV", "fixed/large/slow-XIV")
+
+  def dcrSupportsAllCollectionTypes(faciaPage: PressedPage): Boolean = {
+    faciaPage.collections.forall(collection => SUPPORTED_COLLECTIONS.contains(collection.collectionType))
+  }
+
   def getTier(faciaPage: PressedPage, path: String)(implicit request: RequestHeader): RenderType = {
     val participatingInTest = ActiveExperiments.isParticipating(FrontRendering)
-    val dcrCouldRender = false
+    val dcrCouldRender = dcrSupportsAllCollectionTypes(faciaPage)
+
+    val tier = decideTier(request.forceDCROff, request.forceDCR, participatingInTest, dcrCouldRender)
 
-    val tier = if (request.forceDCROff) {
+    logTier(faciaPage, path, participatingInTest, dcrCouldRender, tier)
+
+    tier
+  }
+
+  def decideTier(
```

Comment:
This is perhaps a Scala thing so take this with a pinch of salt but is there a need for `decideTier`? My reading here is if you can assign as `if` statement to a variable (which Scala seems to let you) then that means you can go a lot further without abstractions?

Reply:
Unfortunately, I had to extract this method. I would also prefer not to do that but if not I have to mock `ActiveExperiments.isParticipating(FrontRendering)` (line 19) in order to control the value of `participatingInTest` to be able to write tests. Mocking this call turned out to be a complicated thing, so decided to create this abstraction. Renamed it from `decideTier` to `getTier` and simplified the if else statements though, I hope this makes sense!

Reply:
Ah, for testing... I see. Fair.